### PR TITLE
Replaced the Dark Souls 3 splitter with the new splitter.

### PR DESCRIPTION
**Pull request by FrankvdStam**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [X] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [X] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [X] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

**Comment by just-ero** [comment]
Why'd you *exclusively* mess up the indentations on some of these? lol

**Comment by FrankvdStam** [comment]
Idk why this is. It shows differently locally. I had the same problem last time

**Comment by just-ero** [comment]
Does it show different locally because you're not displaying leading and trailing white spaces? VSCode has an option to do that, you should enable it, if you're using that. You're using 4-space tab stops... which are the same size as 4 spaces, obviously. That's why they look the same, if you don't show the leading white spaces.

**Comment by FrankvdStam** [comment]
Don't really use VSCode, quickly did this in notepad++. Is it okay like this or does it require an edit?

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -14345,17 +14334,19 @@
     </AutoSplitter>
     <AutoSplitter>
         <Games>
-            <Game>Elden Ring</Game>
+			<Game>Dark Souls III</Game>
+			<Game>Dark Souls 3</Game>
+			<Game>Elden Ring</Game>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:14339]:
```suggestion
            <Game>Dark Souls III</Game>
            <Game>Dark Souls 3</Game>
            <Game>Elden Ring</Game>
```

File: LiveSplit.AutoSplitters.xml [hunk 2]
```diff
@@ -14345,17 +14334,19 @@
...
-        <URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/SoulSplitter.dll</URL>
-            <URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/MaterialDesignColors.dll</URL>
-        <URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/MaterialDesignThemes.Wpf.dll</URL>
-        <URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/MaterialDesignThemes.Wpf.xml</URL>
-            <URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/SoulMemory.dll</URL>
+			<URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/SoulSplitter.dll</URL>
+			<URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/MaterialDesignColors.dll</URL>
+			<URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/MaterialDesignThemes.Wpf.dll</URL>
+			<URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/MaterialDesignThemes.Wpf.xml</URL>
+			<URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/SoulMemory.dll</URL>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:14346]:
```suggestion
            <URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/SoulSplitter.dll</URL>
            <URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/MaterialDesignColors.dll</URL>
            <URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/MaterialDesignThemes.Wpf.dll</URL>
            <URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/MaterialDesignThemes.Wpf.xml</URL>
            <URL>https://raw.githubusercontent.com/FrankvdStam/SoulSplitter/main/Components/SoulMemory.dll</URL>
```

**Comment by FrankvdStam** [comment]
Thanks. Sorry about the indents.

**Pull request merged**